Overview.src.html

```diff
@@ -3391,21 +3456,23 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
   <dl class=switch>
    <dt><code data-anolis-spec=fileapi>Blob</code>
    <dd>
-    <p>Push a copy of <var>object</var>'s contents to <var>stream</var>.
+    <p><span title=concept-enqueue-ReadableStream>Enqueue</span> a <code>Uint8Array</code> object
```

Comment:
missing word "let bytes be the **bytes** read", I think. Maybe "If an exception is thrown" should be "If an exception is thrown creating the ArrayBuffer".

But overall that sounds good to me. It's pretty clear what the intent is.